### PR TITLE
adds WGBH blog post about ingest workflow

### PR DESCRIPTION
**Pull request by ablwr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ If you have never contributed to GitHub before, [here is a guide](http://ablwr.g
 - [University of Michigan ICPSR](http://www.icpsr.umich.edu/icpsrweb/content/datamanagement/preservation/index.html): "This guide describes digital preservation practices at ICPSR, the primary objective of which is to ensure long-term access to the more than 500,000 files in our collection."
 - [University of Washington Media Center](https://github.com/pugetsoundandvision/information/blob/master/UWMC/Open%20Reel%20(Reel%20to%20Reel)%20Tape%20Digitization%20Guide.md): Step by step documentation of open-reel audio digitization workflow in place at the University of Washington Libraries Media Center.
 - [VideoPreservation Website](http://videopreservation.conservation-us.org/): Site to encourage the preservation of historic video using the mature technology of digital capture.
+- [PBS NewsHour Digitization Project Update: Ingest and Digital Preservation Workflows](https://americanarchivepb.wordpress.com/2017/04/12/pbs-newshour-digitization-project-update-ingest-and-digital-preservation-workflows/): Blog post detailing digitization, ingest, and QC workflow from WGBH.
```
Comment (retokromer) [inline README.md]: Why not in alphabetic order?

**Review by retokromer — CHANGES_REQUESTED** [review]

**Comment by ablwr** [comment]
Ack! Sorry! I had originally named it WGBH [...] so I had placed it there.

**Comment by ablwr** [comment]
There we go. :D

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
Thank you!

**Pull request merged**